lib/suspenders/generators/app_generator.rb

```diff
@@ -165,13 +171,20 @@ def install_refills
     end
 
     def setup_git
-      if !options[:skip_git]
+      unless options[:skip_git]
         say 'Initializing git'
         invoke :setup_gitignore
         invoke :init_git
       end
     end
 
+    def git_first_commit
+      unless options[:skip_git]
+        say 'Init commit'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.